product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QXGSKS1u0n8x",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Tour With TensorFlow and Keras"
```

Comment:
General remarks:

- We should definitely divide this tour into two notebooks basic and advanced -> this tour may be the very first contact of people with Neptune and it could be overwhelming 
- We should add some comments into the content to explain things (even at a basic level) -> remember there is also the showcase notebook that could be used in other places

Reply:
done.